testing/src/test/java/io/stargate/it/http/HealthCheckerTest.java

```diff
@@ -36,14 +38,28 @@ public void liveness() throws IOException {
     assertThat(body).isEqualTo("UP");
   }
 
-  @Test
-  public void readiness() throws IOException {
+  @ParameterizedTest
+  @CsvSource({",", "?check=deadlocks", "?check=graphql", "?check=deadlocks&check=graphql"})
+  public void readiness(String query) throws IOException {
+    query = query == null ? "" : query;
     String body =
-        RestUtils.get("", String.format("%s:8084/checker/readiness", host), HttpStatus.SC_OK);
+        RestUtils.get(
+            "", String.format("%s:8084/checker/readiness%s", host, query), HttpStatus.SC_OK);
 
     assertThat(body).isEqualTo("READY");
   }
 
+  @Test
+  public void missingReadinessCheck() throws IOException {
+    String body =
+        RestUtils.get(
+            "",
+            String.format("%s:8084/checker/readiness?check=testUnknown", host),
+            HttpStatus.SC_SERVICE_UNAVAILABLE);
+
+    assertThat(body).isEqualTo("NOT READY");
+  }
+
```

Comment:
missing test case for a known health check that reports unhealthy.

Reply:
added unit tests